auth-api/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>stargate</artifactId>
-    <version>1.0.5-SNAPSHOT</version>
+    <version>1.0.5</version>
```

Comment:
Hmmm, does not look right :thinking:

Reply:
Yeah it does that when you tag and then retag. I'll fix it